src/main/java/com/google/cloud/tools/gradle/appengine/core/DeployExtension.java

```diff
@@ -77,7 +77,9 @@ public void setBucket(String bucket) {
     return deployables;
   }
 
-  public void setDeployables(Object deployables) {
+  // The name of this method is no longer "setDeployables" so the user can't configure it in
+  // their build file.
+  public void updateDeployables(Object deployables) {
```

Comment:
Since this will just be an internal method, the parameter type can probably be a list of files.